src/network/networkd-wifi.c

```diff
@@ -90,6 +90,21 @@ int manager_genl_process_nl80211_config(sd_netlink *genl, sd_netlink_message *me
 
         r = link_get_by_index(manager, ifindex, &link);
         if (r < 0) {
+                /* The NL80211_CMD_NEW_INTERFACE message might arrive before RTM_NEWLINK, in which case a
+                 * link will not have been created yet. Store the interface index such that the wireless
+                 * properties of the link (such as wireless interface type) are queried again after the link
+                 * is created.
+                 */
+                if (cmd == NL80211_CMD_NEW_INTERFACE) {
+                        r = set_ensure_put(&manager->new_wlan_links, NULL, INT_TO_PTR(ifindex));
+                        if (r < 0)
+                                log_error_errno(r, "Failed to add new wireless interface index to set: %m");
```

Comment:
Please downgrade the log level (maybe to warning), and mention that the failure will be ignored (i.e., append `", ignoring"` like the other log messages).